spec/features/global_chat_spec.rb

```diff
@@ -35,17 +30,42 @@ def post_message(message)
     end
   end
 
-  it 'shows the correct timestamp' do
-    post_message(message)
-    message = RoomMessage.all.last
-    expect(page).to have_text(message.formatted_time)
-  end
+  context 'with a posted message' do
+    let(:room_message) { RoomMessage.all.last }
 
-  it 'shows the correct timestamp after reload' do
-    post_message(message)
-    message = RoomMessage.all.last
-    visit root_path
-    expect(page).to have_text(message.formatted_time)
+    before do
+      post_message(message)
+      room_message
+    end
+
+    it 'clears text box after message submit' do
+      expect(page).to have_field('room_message_message', text: '')
+    end
+
+    it 'shows the correct timestamp' do
+      within('#chat-messages') do
+        expect(page).to have_text(room_message.formatted_time)
+      end
+    end
+
+    it 'links to the users profile page' do
```

Comment:
We would need two more specific tests for that because the user name (do we test if that is only displayed for messages from other users?) and the profile picture both have the same link.

With so many changes to the tests, we could really open a new PR (if you did not start with the tests yet)